lib/reek/context/ghost_context.rb

```diff
@@ -8,6 +8,8 @@ module Context
     # context tree. This context will not be part of the resulting tree, but
     # will track context and visibility separately while building is in
     # progress.
+    #
+    # :reek:InstanceVariableAssumption
```

Comment:
Due to the changes in the detector, this class no longer has this smell so these lines can be removed.